SwiftCompilerSources/Sources/Optimizer/FunctionPasses/ObjectOutliner.swift

```diff
@@ -403,8 +421,9 @@ private extension FunctionPassContext {
 
 private func optimizeFindStringCall(stringArray: GlobalValueInst, _ context: FunctionPassContext) {
   if stringArray.numArrayElements > 16,
-     let findStringCall = findFindStringCall(stringArray: stringArray),
-     let cachedFindStringFunc = getFindStringSwitchCaseWithCacheFunction(context) {
+    let findStringCall = findFindStringCall(stringArray: stringArray),
```

Comment:
I find it more readable if additional if-conditions in succeeding lines are aligned to the first condition in the first line:
```
if let ...,
   let ...           <- three spaces
{
```